tests/integration/actions/test_inventory.py

```diff
@@ -7,7 +7,7 @@
 from ... import defaults
 from .._common import get_executable_path
 from .._common import fixture_path_from_request
-from .._common import update_fixtures
+from .._common import update_fixtures  # pylint: disable=unused-import
```

Comment:
Is this change necessary with the `UPDATE_FIXTURES` changes? Can we just leave it uncommented and conditionalized on the value of the constant?